samples/guestbook/README.md

```diff
@@ -96,26 +96,29 @@ directory using the same variables you entered during `terraform apply`.
 ## Running on Amazon Web Services (AWS)
 
 If you want to run this sample on AWS, you need to set up an account, download
-the AWS command line interface, and log in. You will also need an SSH key. If
-you don't already have one, you can follow [this guide from GitHub][GitHub SSH].
-Follow the instructions for "Adding your key to the ssh-agent" if you want the
-key to persist across terminal sessions.
+the AWS command line interface, and log in. There's [help here][AWS Config Help]
+if you need it.
 
-### Agree to the AWS Terms and Conditions
```

Comment:
This is important actually; please keep this section. We don't hit it anymore because we've already consented to the Debian T&C.

Reply:
I went to the linked page, including in an incognito window, and didn't see anywhere to agree to T&C....?

Reply:
One has to click "Continue to Subscribe" which then will prompt for it.

Reply:
OIC. Added it back, and added a sentence about how to do it.